### PR TITLE
NE-704: add NLB usage metrics for network edge

### PR DESCRIPTION
**Pull request by tjungblu**
* [x] I added CHANGELOG entry for this change.
* [ ] No user facing changes, so no entry in CHANGELOG was needed.

exposing the metric from https://github.com/openshift/cluster-monitoring-operator/pull/1509

### CODE REVIEW

**Comment by tjungblu** [comment]
/hold 
until we merge the other PR :)

**Review by simonpasquier — COMMENTED** [review]
Looking at https://github.com/openshift/cluster-ingress-operator/pull/675 the metric has a `name` label. What could be the cardinality of that label?

**Comment by tjungblu** [comment]
@simonpasquier yeah good catch, I don't think that would even be necessary. Let me patch that out of the metric.

File: jsonnet/rules.libsonnet [hunk 1]
```diff
@@ -513,6 +513,11 @@ function(params) {
           record: 'cluster:usage:openshift:ingress_request_total:irate5m',
           // The instantaneous rate of openshift requests per second arriving at the ingress controllers
         },
+        {
+          expr: 'sum(ingress_controller_aws_nlb_active) or absent(__does_not_exist__)*0',
```
Comment (simonpasquier) [inline jsonnet/rules.libsonnet:517]:
```suggestion
          expr: 'sum(ingress_controller_aws_nlb_active) or vector(0)',
```

**Comment by simonpasquier** [comment]
/lgtm

/label px-approved
/label qe-approved
/label docs-approved

PX, docs and QE have alreay approved in https://issues.redhat.com/browse/NE-704. Given that it's only adding 1 series per cluster, I expect that @smarterclayton is ok with it so I'm not holding the PR.

**Pull request merged**